src/scalar_impl.h

```diff
@@ -254,6 +254,14 @@ static void secp256k1_scalar_inverse_var(secp256k1_scalar *r, const secp256k1_sc
 
 #ifdef USE_ENDOMORPHISM
 #if defined(EXHAUSTIVE_TEST_ORDER)
+#  if EXHAUSTIVE_TEST_ORDER == 13
+#    define EXHAUSTIVE_TEST_LAMBDA 9
+#  elif EXHAUSTIVE_TEST_ORDER == 199
+#    define EXHAUSTIVE_TEST_LAMBDA 92
+#  else
+#    error No known lambda for the specified exhaustive test group order.
+#endif
```

Comment:
nit: indentation.

No lambda for order 7?

Reply:
I just pushed another update.

The order 7 subgroup in fact does not have the endomorphism property (apparently all subgroups do, except the order 2, 3, and 7 ones).

Reply:
Ok I see.

Can you fix the #endif indentation?

Reply:
It's already fixed, I think?

Reply:
Oh, it was fixed in a later commit only. Really fixed now.

Reply:
Oh sorry, I thought I looked in the later commits too. For some reason I missed it there.